vxdev/update-code.sh

```diff
@@ -49,8 +53,26 @@ elif [[ $BRANCH == 'stable' ]]; then
 	cd vxsuite
 	./script/setup-dev
 	cd ..
+elif [[ $BRANCH == 'custom' ]]; then
+	read -p "Enter the branch name: " BRANCH_NAME
+	cd vxsuite
+	git checkout main
+	git pull
+	if [ !`git branch -r --list origin/$BRANCH_NAME` ]
```

Comment:
Does this work? If you want to redirect the output I suggest doing this:

```suggestion
	if ! git branch -r --list origin/$BRANCH_NAME > /dev/null 2>&1
```

Reply:
Yep, this does work fwiw